flask/main.py

```diff
@@ -97,15 +97,15 @@ def products():
         raise(err)
 
     try:
-        r = requests.get(RUBY_BACKEND + "/api")
+        r = requests.get(RUBY_BACKEND + "/api", headers=request.headers)
```

Comment:
@sergiosentry let me think this through. I'm concerned if we re-use the same headers, then we'll lose any headers that the requests.get request was going to set itself, as well as any request headers that App Engine (the production environment) will be adding. So I'd at least need to test this in staging first.

Basically the request headers on the Ruby transaction would not longer be the real request headers sent by Flask. Instead, they'd be the request headers sent from React-to-Flask. Does that make sense?

It might be better to set se, customerType and email one-by-one like:
```
headers={se, customerType, email}
```
and then I think all the other headers by the requests library and AppEngine would still get set. I'm just concerned we'd be overriding them all if we did `headers=request.headers`. Let me get some opinions on this.

Reply:
![image](https://user-images.githubusercontent.com/8920574/181212605-8db2c19c-c985-4845-b050-84b549b737d7.png)